Common/Tests/Helper/AuthenticationTokenGeneratorMock.swift

```diff
@@ -9,6 +9,8 @@ struct AuthenticationTokenGeneratorMock {
     private let insightsUrl: String?
     private let environment: String?
 
+    static var isInsightsConfigValid = true
```

Comment:
I dont think this is related to Insights - name should be `isConfigValid`

Reply:
right :)

Reply:
fixed